### PR TITLE
I have a problem understanding how to modify header. So I fix the document

### PR DESCRIPTION
**Pull request by pacozaa**
- Default of `outerContainerStyles`  and `innerContainerStyles` is not `none` as the document state.

-  I give and example how to manage `innerContainerStyles` which is important

### CODE REVIEW

File: docs/API/header.md [hunk 1]
```diff
@@ -82,5 +84,5 @@ We wanted the Header to be as customisable as possible, so you are free to try d
 | centerComponent | none | configuration object for default component (text: string, ...props for React Native Text component) valid React Element | define your center component here |
 | rightComponent | none | configuration object for default component (icon: string, ...props for React Native Elements Icon component) or a valid React Element | define your right component here |
 | backgroundColor | none | string | sets backgroundColor of the parent component |
-| outerContainerStyles | none | object (style) | styling for outer container |
-| innerContainerStyles | none | object (style) | styling for inner container |
+| outerContainerStyles | position: 'absolute',<br>top: 0,<br>left: 0,<br>right: 0,<br>backgroundColor: '#fff',<br>borderBottomColor: '#f2f2f2',<br>borderBottomWidth: 1,<br>padding: 15,<br>height: 70 | object (style) | styling for outer container |
```
Comment (iRoachie) [inline docs/API/header.md]: It may be easier and clearer to instead use a link to the definition in the source

File: docs/API/header.md [hunk 2]
```diff
@@ -82,5 +84,5 @@ We wanted the Header to be as customisable as possible, so you are free to try d
...
+| innerContainerStyles | flex: 1,<br>flexDirection: a'row',<br>justifyContent: 'space-between',<br>alignItems: 'flex-end' | object (style) | styling for inner container |
```
Comment (iRoachie) [inline docs/API/header.md]: Same as previous

File: docs/API/header.md [hunk 3]
```diff
@@ -62,17 +62,19 @@ Components defined through props take precedence over components passed in as ch
...
 <Header
   statusBarProps={{ barStyle: 'light-content' }}
   leftComponent={<MyCustomLeftComponent />}
   centerComponent={{ text: 'MY TITLE', style: { color: '#fff'} }}
   outerContainerStyles={{ backgroundColor: '#3D6DCC' }}
+  innerContainerStyles={{ justifyContent: 'space-around',}}  
 />
 ```
 
+
```
Comment (iRoachie) [inline docs/API/header.md]: Extra line?

File: docs/API/header.md [hunk 4]
```diff
@@ -62,17 +62,19 @@ Components defined through props take precedence over components passed in as ch
 
 ### Header customisability
 
-We wanted the Header to be as customisable as possible, so you are free to try different combinations of props.
+We wanted the Header to be as customisable as possible, so you are free to try different combinations of props. For example if you want to change left,center,right component's layout, try to play with ` innerContainerStyles `
```
Comment (iRoachie) [inline docs/API/header.md]: Maybe something more friendly?

For example, if you want to change the left, center, or right component's layout, you can adjust the `innerContainerStyles`

**Comment by pacozaa** [comment]
I will do it :)

**Pull request closed without merging**